arcgis-ios-sdk-samples/Layers/Map image layer tables/MapImageLayerTablesViewController.swift

```diff
@@ -0,0 +1,158 @@
+        alertController.addAction(cancelAction)
+        // Present the controller.
+        alertController.popoverPresentationController?.barButtonItem = queryBarButtonItem
+        present(alertController, animated: true)
+    }
+    
+    /// Make a map for the map view.
+    func makeMap() -> AGSMap {
+        // Create a map with the ArcGIS streets basemap style.
+        let map = AGSMap(basemapStyle: .arcGISStreets)
+        // Add the layer to the map.
+        map.operationalLayers.add(serviceRequestsMapImageLayer)
+        // Load the map image layer's tables and layers.
+        serviceRequestsMapImageLayer.loadTablesAndLayers { [weak self] _ in
+            self?.queryFeatures()
+        }
+        return map
+    }
+    
+    /// Make a renderer for the graphics overlay.
+    func makeRenderer () -> AGSSimpleRenderer {
+        let renderer = AGSSimpleRenderer()
+        // Create a symbol for the feature.
+        let selectedRequestSymbol = AGSSimpleMarkerSymbol(style: .circle, color: .cyan, size: 14)
+        // Set the renderer's symbol.
+        renderer.symbol = selectedRequestSymbol
+        return renderer
+    }
+    
+    /// Query features on the first table in the map image layer.
+    func queryFeatures() {
+        // Create query parameters and set its where clause.
+        let nullCommentsParameters = AGSQueryParameters()
+        nullCommentsParameters.whereClause = "requestid <> '' AND comments <> ''"
+        // Set the first table from the map image layer.
+        commentsTable = serviceRequestsMapImageLayer.tables.first
+        // Query features on the feature table with the query parameters and all feature fields.
+        commentsTable?.queryFeatures(with: nullCommentsParameters, queryFeatureFields: .loadAll) { [weak self] result, error in
+            guard let self = self else { return }
+            if let comments = result?.featureEnumerator().allObjects {
+                // Show the records from the service request comments table in the list view control.
+                self.commentsArray = comments
+                // Enable the button after the map and features have been loaded.
+                self.queryBarButtonItem.isEnabled = true
+            } else if let error = error {
+                self.presentAlert(error: error)
+            }
+        }
+    }
+    
+    /// Query related features for the selected feature.
+    func queryCommentsTable(feature: AGSArcGISFeature) {
+        // Get the relationship that defines related service request features for features in the comments table (this is the first and only relationship).
+        let commentsRelationshipInfo = commentsTable?.layerInfo?.relationshipInfos.first
+        // Create query parameters to get the related service request for features in the comments table.
+        let relatedQueryParameters = AGSRelatedQueryParameters(relationshipInfo: commentsRelationshipInfo!)
```

Comment:
```suggestion
        // Get the relationship that defines related service request features for features in the comments table (this is the first and only relationship).
        guard let commentsRelationshipInfo = commentsTable?.layerInfo?.relationshipInfos.first else { return }
        // Create query parameters to get the related service request for features in the comments table.
        let relatedQueryParameters = AGSRelatedQueryParameters(relationshipInfo: commentsRelationshipInfo)
```
Consider guard it so that it doesn't need to be force unwrapped on next line.